internal/docs/rest/rest.go

```diff
@@ -0,0 +1,248 @@
+)
+
+// GenTree generates the docs for the full tree of commands
+func GenTree(cmd *cobra.Command, dir string) error {
+	for _, c := range cmd.Commands() {
+		if !c.IsAvailableCommand() || c.IsAdditionalHelpTopicCommand() {
+			continue
+		}
+		if err := GenTree(c, dir); err != nil {
+			return err
+		}
+	}
+
+	basename := strings.ReplaceAll(cmd.CommandPath(), " ", separator) + defaultExtension
+	filename := filepath.Join(dir, basename)
+	f, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	if err := GenCustom(cmd, f); err != nil {
+		return err
+	}
+	return nil
+}
+
+const toc = `
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+`
+
+const syntaxHeader = `Syntax
+------
+
+.. code-block::
+`
+
+const examplesHeader = `Examples
+--------
+
+.. code-block::
+`
+
+const tocHeader = `
+.. toctree::
+   :titlesonly:
+`
+
+// GenCustom creates custom reStructured Text output.
+// Adapted from github.com/spf13/cobra/doc to match MongoDB tooling and style
+func GenCustom(cmd *cobra.Command, w io.Writer) error {
+	cmd.InitDefaultHelpCmd()
+	cmd.InitDefaultHelpFlag()
+
+	buf := new(bytes.Buffer)
+	name := cmd.CommandPath()
+
+	ref := strings.ReplaceAll(name, " ", separator)
+
+	buf.WriteString(".. _" + ref + ":\n\n")
+	buf.WriteString(strings.Repeat("=", len(name)) + "\n")
+	buf.WriteString(name + "\n")
+	buf.WriteString(strings.Repeat("=", len(name)) + "\n")
+	buf.WriteString(toc)
+	buf.WriteString("\n" + cmd.Short + "\n")
+	if cmd.Long != "" {
+		buf.WriteString("\n" + cmd.Long + "\n")
+	}
+	buf.WriteString("\n")
+
+	if cmd.Runnable() {
+		buf.WriteString(syntaxHeader)
+		buf.WriteString(fmt.Sprintf("\n   %s\n\n", strings.ReplaceAll(cmd.UseLine(), "[flags]", "[options]")))
+	}
+	printArgsReST(buf, cmd)
+	printOptionsReST(buf, cmd)
+
+	if len(cmd.Example) > 0 {
+		buf.WriteString(examplesHeader)
+		buf.WriteString(fmt.Sprintf("\n%s\n\n", indentString(cmd.Example, " ")))
+	}
+
+	if hasRelatedCommands(cmd) {
+		buf.WriteString("Related Commands\n")
+		buf.WriteString("--------\n\n")
+
+		children := cmd.Commands()
+		sort.Sort(byName(children))
+
+		for _, child := range children {
+			if !child.IsAvailableCommand() || child.IsAdditionalHelpTopicCommand() {
+				continue
+			}
+			cname := name + " " + child.Name()
+			ref = strings.ReplaceAll(cname, " ", separator)
+			buf.WriteString(fmt.Sprintf("* :ref:`%s` - %s\n", ref, child.Short))
+		}
+		buf.WriteString("\n")
+	}
+	if _, ok := cmd.Annotations["toc"]; ok || !cmd.Runnable() {
+		buf.WriteString(tocHeader)
+		buf.WriteString("\n")
+		children := cmd.Commands()
+		sort.Sort(byName(children))
+
+		for _, child := range children {
+			if !child.IsAvailableCommand() || child.IsAdditionalHelpTopicCommand() {
+				continue
+			}
+			cname := name + " " + child.Name()
+			ref = strings.ReplaceAll(cname, " ", separator)
+			buf.WriteString(fmt.Sprintf("   /reference/%s\n", ref))
```

Comment:
Would it be possible to produce a list of child commands like this?
```
create </reference/mongocli-atlas-accessLists-create>
delete </reference/mongocli-atlas-accessLists-delete>
describe </reference/mongocli-atlas-accessLists-describe>
list </reference/mongocli-atlas-accessLists-list>
```
This would greatly reduce clutter in the left nav. See a [staged example](https://docs-mongodbcom-staging.corp.mongodb.com/mongocli/docsworker-xlarge/gen-docs-test/reference/mongocli-atlas-accessLists/).
I realize this is a late request so we don't have to do it in this PR, just wondering if its possible!

Reply:
done